pkg/service/utgen/assets/test_generation.toml

```diff
@@ -128,30 +212,30 @@ new_tests:
 {%- endif %}
   test_code: |
 {%- if language in ["python"] %}
+    @pytest.mark.asyncio
     def ...
 {%- else %}
     ...
 {%- endif %}
   new_imports_code: |
     {%- if language == "python" %}
-    "import pytest"
-    "from my_module import my_function"
+    import pytest
     {%- elif language == "java" %}
-    "import org.junit.jupiter.api.Test;"
-    "import my.package.MyFunction;"
+    import org.junit.jupiter.api.Test;
+    import my.package.MyFunction;
     {%- elif language == "golang" %}
-    "import "testing" "
-    "import "my_module""
+    import "testing"
+    import "my_module"
     {%- elif language == "javascript" %}
-    "const assert = require('assert');"
-    "const myFunction = require('my_module').myFunction;"
+    const assert = require('assert');
+    const myFunction = require('my_module').myFunction;
     {%- elif language == "typescript" %}
-    "import { assert } from 'assert';"
-    "import { myFunction } from 'my_module';"
+    import { assert } from 'assert';
+    import { myFunction } from 'my_module';
     {%- endif %}
   library_installation_code: |
     {%- if language == "python" %}
-    pip install pytest
+    pip install pytest pytest-asyncio
     {%- elif language == "java" %}
```

Comment:
this was a sampler any reason of adding asyncio, if we remove it would it cause any issue?

Reply:
Ya that was just for the decorator, I handled it below, will remove it